pages/pipelines/environment_variables.md.erb

```diff
@@ -122,18 +202,30 @@ The following environment variables are automatically provided to every job, and
     <th><code>BUILDKITE_REBUILT_FROM_BUILD_NUMBER</code></th>
     <td>The UUID of the original build this was rebuilt from. <p class="Docs__api-param-eg"><em>Example:</em> <code>"1514"</code>, or <code>""</code> if not a rebuild.</p></td>
   </tr>
+  <tr>
+    <th><code>BUILDKITE_REFSPEC</code></th>
+    <td>A custom refspec for the <code>buildkite-agent</code> bootstrap script to use when checking out code. <p class="Docs__api-param-eg"><em>Example:</em> <code>"+refs/weird/123abc:refs/local/weird/456"</code></p></td>
+  </tr>
   <tr>
     <th><code>BUILDKITE_REPO</code></th>
     <td>The repository of your pipeline. <p class="Docs__api-param-eg"><em>Example:</em> <code>"git@github.com:acme-inc/my-project.git"</code></p></td>
   </tr>
   <tr>
-    <th><code>BUILDKITE_REFSPEC</code></th>
-    <td>A custom refspec for the <code>buildkite-agent</code> bootstrap script to use when checking out code. <p class="Docs__api-param-eg"><em>Example:</em> <code>"+refs/weird/123abc:refs/local/weird/456"</code></p></td>
+    <th><code>BUILDKITE_RETRY_COUNT</code></th>
+    <td>An integer of how many times this job has been retried. <p class="Docs__api-param-eg"><em>Example:</em> <code>"0"</code></p></td>
```

Comment:
Oh wow, this is one of the big standard ones that is missing! We should add this straight away perhaps?

```suggestion
    <td>How many times this job has been retried. <p class="Docs__api-param-eg"><em>Example:</em> <code>"0"</code></p></td>
```